.github/workflows/desktop-file-check.yaml

```diff
@@ -3,7 +3,7 @@ on:
   workflow_dispatch: # Manual trigger
   pull_request:
     paths:
-      - qt/res/OrganicMaps.desktop
+      - qt/res/app.organicmaps.desktop.desktop
```

Comment:
Yes exactly, because the reverse DNS name is `app.organicmaps.desktop` + the `.desktop` suffix.

Reply:
I assume it may break a lot of links, right? We discussed it 2 years ago.

Reply:
> Why don't have "app.organicmaps" app name instead?

Well, AFAIK  `app.organicmaps` alone for app-id would be invalid, because that has only two parts, and the specifications mandate three parts.

> I assume it may break a lot of links, right? We discussed it 2 years ago.

Provided that there is a valid 3 part alternative to `app.organicmaps.desktop` that the project would agree on, then  
there would be:
1. A considerable amount of chore to sunset the [app.organicmaps.desktop](https://github.com/flathub/app.organicmaps.desktop) package, set up and validate the new one, and also mark the new one as a successor to the old. Optionally provide a migration guide for a potentially sizable local data, if we care about that.
2. ~ 10 -is [links to change here](https://github.com/search?q=repo%3Aorganicmaps%2Forganicmaps.github.io+app.organicmaps.desktop&type=code) ~ 5 is [renaming here](https://github.com/search?q=repo%3Aorganicmaps%2Forganicmaps%20app.organicmaps.desktop&type=code)
3. Anything else?

Reply:
We can put our time to more important tasks, no? )